tests/printer/expr/asyncAwait.res

```diff
@@ -98,4 +98,7 @@ let f12 = @a (@b x) => 3
 let f13 = @a @b (~x) => 3
 
 let aw = (await (server->start))->foo
-let aw = (@foo (server->start))->foo
\ No newline at end of file
+let aw = (@foo (server->start))->foo
+
+let foo = async(~a=34)
```

Comment:
Might be more appropriate in the `tests/parsing/grammar/expressions/async` directory as it's really about parsing

Reply:
Did that first but wanted to check that it does not mess up printing.
Probably both.